cybench/runs/run_benchmark.py

```diff
@@ -40,41 +40,23 @@
 _BASELINE_MODEL_INIT_KWARGS = defaultdict(dict)
 _BASELINE_MODEL_INIT_KWARGS["LinearTrend"] = {"trend": "linear"}
 
-_BASELINE_MODEL_INIT_KWARGS["SklearnRidge"] = {"sklearn_est": sklearn_ridge}
-_BASELINE_MODEL_INIT_KWARGS["SklearnRF"] = {"sklearn_est": sklearn_rf}
-
 _BASELINE_MODEL_INIT_KWARGS["LSTM"] = {
     "hidden_size": 64,
     "num_layers": 1,
 }
 
 _BASELINE_MODEL_FIT_KWARGS = defaultdict(dict)
-
 _BASELINE_MODEL_FIT_KWARGS["SklearnRidge"] = {
+    "select_features": True,
+    "optimize_hyperparameters": True,
+}
+
+_BASELINE_MODEL_FIT_KWARGS["SklearnRF"] = {
     "optimize_hyperparameters": True,
-    "param_space": {"estimator__alpha": [0.01, 0.1, 0.0, 1.0, 5.0, 10.0]},
 }
 
 _BASELINE_MODEL_FIT_KWARGS["LSTM"] = {
```

Comment:
Looks like device is not specified in current LSTM fit kwargs, defaulting it to CPU. Should be added to default fit kwargs.

Reply:
It's addressed in latest changes.